readme.txt

```diff
@@ -2,7 +2,7 @@
 Contributors: mayukojpn, nao, dartui, pedromendonca, casiepa, mekemoke, miyauchi, nekojonez
 Tags: translation
 Requires at least: 4.9
-Tested up to: 6.0
+Tested up to: 6.0.2
```

Comment:
No need to bump the point release, should be left as 6.0, as there are no breaking changes in point releases.